extensions/http-status-codes/src/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ActionPanel, Action, Icon, List } from "@raycast/api";
+import codes from "./codes.json";
+import React from "react";
+
+type Code = {
+  code: string;
+  description: string;
+  reference: string;
+};
+
+export default function Command() {
+  const codeGroups = (codes as Code[]).reduce((groups: { [firstDigit: string]: Code[] }, code) => {
+    const group = groups[code.code[0]] || [];
+    group.push(code);
+    groups[code.code[0]] = group;
+
+    return groups;
+  }, {});
+
+  return (
+    <List isLoading={false} searchBarPlaceholder="Filter by code or description...">
+      {Object.entries(codeGroups).map(([firstDigit, codes]) => (
+        <List.Section key={firstDigit} title={`${firstDigit}xx`} subtitle={getCodeGroupDescription(firstDigit)}>
+          {codes.map((code) => (
+            <List.Item
+              key={code.code}
+              title={code.code}
+              subtitle={code.description}
+              keywords={[code.description]} // make subtitle searchable
+              icon={{
+                source: Icon.Dot,
+                tintColor: statusCodeToColor(code.code),
+              }}
+              accessoryTitle={code.reference}
+              actions={
+                <ActionPanel>
+                  <Action.OpenInBrowser
+                    title="Open in browser"
+                    url={`https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/${code.code}`}
```

Comment:
Not all status codes listed here are on MDN 👀 
For instance, (for some reason 😄) I remember 102 isn't https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/102

Reply:
Good point! Looks like using `require("http").STATUS_CODES` would fix that too :D